test/evergreen.yml

```diff
@@ -238,6 +238,13 @@ tasks:
         vars:
           posix_configure_flags: --enable-strict --enable-diagnostic
 
+  - name: compile-enable-strict
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: --enable-strict
+
```

Comment:
There's nothing special for this configure flag setting compared to the default. I'd prefer to reuse the default `compile` task instead of creating a new one. I know it's safer to do an apple-to-apple translation and could understand if you prefer to keep this way.

Reply:
I can either reuse "compile-enable-strict" task in "checkpoint-filetypes-test" which compiles only with --enable-strict or use default compile as you suggested.

Reply:
Using the default compile task.